### PR TITLE
fix: update containerd config

### LINKED ISSUE #121: containerd config need update
maybe use `containerd config default > /etc/containerd/config.toml` auto generate

/kind optimize

### PR DESCRIPTION
**Pull request by lyp256**
Signed-off-by: lyp256 <lyp256@qq.com>

### What type of PR is this?

/kind bug

### What this PR does / why we need it:
containerd uses an outdated configuration file
### Which issue(s) this PR fixes:

Fixes #121

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs
```

### CODE REVIEW

**Comment by x893675** [comment]
@lyp256  fix ci check first

File: pkg/scheme/core/v1/cri/containerd.go [hunk 1]
```diff
@@ -256,3 +261,166 @@ func (runnable *ContainerdRunnable) renderTo(w io.Writer) error {
 	_, err := at.RenderTo(w, configTomlTemplate, runnable)
 	return err
 }
+
+func (runnable *ContainerdRunnable) renderRegistryConfig(dryRun bool) error {
+	if dryRun {
+		return nil
+	}
+	configDir := containerdDefaultRegistryConfigDir
+	if runnable.RegistryConfigDir != "" {
```
Comment (lixd) [inline pkg/scheme/core/v1/cri/containerd.go:270]: where is  RegistryConfigDir from? Does front add a field？

File: pkg/scheme/core/v1/cri/template.go [hunk 2]
```diff
@@ -42,153 +42,261 @@ const daemonConfigTemplate = `{
...
+        runtime_root = ""
+        runtime_type = ""
+
+        [plugins."io.containerd.grpc.v1.cri".containerd.untrusted_workload_runtime.options]
+
+    [plugins."io.containerd.grpc.v1.cri".image_decryption]
+      key_model = "node"
+
     [plugins."io.containerd.grpc.v1.cri".registry]
+      config_path = ""
```
Comment (lixd) [inline pkg/scheme/core/v1/cri/template.go:197]: is this need to sync with  runnable.RegistryConfigDir.
like

File: pkg/scheme/core/v1/cri/containerd.go (hunk 1 above)
Comment (lyp256) [reply pkg/scheme/core/v1/cri/containerd.go:270]: This field is currently reserved to enhance customizability, and can be obtained from the web page later

**Review by lixd — APPROVED** [review]
/lgtm

**Review by lixd — APPROVED** [review]
/lgtm

**Comment by lixd** [comment]
tested,it works, can pull images from HTTPS repo.

**Pull request merged**